sagemaker-python-sdk/tensorflow_script_mode_mnist/tensorflow_distributed_mnist.ipynb

```diff
@@ -0,0 +1,224 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# MNIST distributed training  \n",
+    "\n",
+    "The **SageMaker Python SDK** helps you deploy your models for training and hosting in optimized, productions ready containers in SageMaker. The SageMaker Python SDK is easy to use, modular, extensible and compatible with TensorFlow and MXNet. This tutorial focuses on how to create a convolutional neural network model to train the [MNIST dataset](http://yann.lecun.com/exdb/mnist/) with a new SageMaker supported format which is called **Script Mode**.\n",
```

Comment:
I don't think "SageMaker Python SDK" needs to be bold. Also we're training a model, not a dataset.